### PR TITLE
new documentation

### PR DESCRIPTION
**Pull request by therealbobo**
Signed-off-by: Roberto Scolaro <roberto.scolaro21@gmail.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

/kind cleanup

> /kind design

/kind documentation

> /kind failing-test

> /kind feature

> /kind flaky-test

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area library

> /area cli

> /area tests

> /area examples

**What this PR does / why we need it**:

The current README.md is outdated.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
NONE

### CODE REVIEW

**Review by loresuso — COMMENTED** [review]
This is great! Just wanted to leave further ideas and suggestions to improve the documentation. It would be nice to 
- have an example of a *full* configuration file for falcoctl
- have a list of all the environment variables that can be used. Although they are pretty schematic (maybe autogenerate it?), the content is something that we defined in a custom way (see for instance the one for authentication methods). I would add this to the docs only, and maybe just one example of their use in long explanation of commands. What about a simple two column table for that?

| name | content |
|------|---------|
| FALCOCTL_REGISTRY_AUTH_BASIC | registry,username,password;registry1,username1,password1|
| ... | ... |

Last nit: I would prepend a `$` to each command that we show in the README.md file, so that it better gives the idea of a prompt. 

Thank you!

File: README.md [hunk 1]
```diff
@@ -115,6 +109,30 @@ Install the *cloudtrail-rules* rulesfile:
...
+    every: 6h0m0s
+    falcoVersions: http://localhost:8765/versions
+    refs:
+    - falco-rules:0
+    - my-rules:1
+indexes:
+- name: falcosecurity
+  url: https://falcosecurity.github.io/falcoctl/index.yaml
+- name: my-index
+  url: https://example.com/falcoctl/index.yaml
```
Comment (loresuso) [inline README.md:132]: I would like here to show every parameter that is configurable, so putting also the section for authentication e install. What do you think?

File: README.md [hunk 2]
```diff
@@ -204,46 +222,87 @@ It shows the OCI **reference** and **tags** for the **artifact** of interest. Th
...
+| `FALCOCTL_ARTIFACT_INSTALL_REFS` | `ref1;ref2` |
+| `FALCOCTL_ARTIFACT_INSTALL_RULESFILEDIR` | `rules-directory-path` |
+| `FALCOCTL_ARTIFACT_INSTALL_PLUGINSDIR` | `plugins-directory-path` |
+
+Please note that when passing multiple arguments via an environment variable, they must be separated by a semicolon. Moreover, multiple fields of the same argument must be separated by a comma.
+
+Here is an example of `falcoctl` usage with environment variables:
+
+```bash
+$ export FALCOCTL_REGISTRY_AUTH_BASIC="localhost:6000;000000;999999;http://localhost:9096/token"
```
Comment (loresuso) [inline README.md:305]: it should be FALCOCTL_REGISTRY_AUTH_OAUTH here :)

File: README.md (hunk 1 above)
Comment (therealbobo) [reply README.md:132]: Absolutely! It's a great idea!

**Review by loresuso — APPROVED** [review]
/approve

**Comment by loresuso** [comment]
cc @leogr

**Review by leogr — APPROVED** [review]

**Pull request merged**